Traceback (most recent call last):
  File "/Users/junaluzi/opt/anaconda3/envs/quant-econ-book/lib/python3.8/site-packages/jupyter_cache/executors/basic.py", line 157, in execute
    executenb(nb_bundle.nb, cwd=tmpdirname)
  File "/Users/junaluzi/opt/anaconda3/envs/quant-econ-book/lib/python3.8/site-packages/nbconvert/preprocessors/execute.py", line 737, in executenb
    return ep.preprocess(nb, resources, km=km)[0]
  File "/Users/junaluzi/opt/anaconda3/envs/quant-econ-book/lib/python3.8/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/Users/junaluzi/opt/anaconda3/envs/quant-econ-book/lib/python3.8/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/Users/junaluzi/opt/anaconda3/envs/quant-econ-book/lib/python3.8/site-packages/nbconvert/preprocessors/execute.py", line 448, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, out)
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
// This is duplicated as a way to use glue functionality
fig, ax = plt.subplots()
plt.plot(Ïµ_values)
plt.show()
------------------

[0;36m  File [0;32m"<ipython-input-3-3aca4e5dd00b>"[0;36m, line [0;32m1[0m
[0;31m    // This is duplicated as a way to use glue functionality[0m
[0m    ^[0m
[0;31mSyntaxError[0m[0;31m:[0m invalid syntax

SyntaxError: invalid syntax (<ipython-input-3-3aca4e5dd00b>, line 1)

